kitchen-tests/.kitchen.travis.yml

```diff
@@ -43,16 +42,6 @@ platforms:
       - RUN /usr/bin/apt-get update
       - RUN /usr/bin/apt-get -y install zlib1g-dev sudo net-tools wget ca-certificates
 
-- name: centos-5
```

Comment:
We weren't actually using this in the config. It was just defined in the Kitchen yaml.